tests/ts/rapunzel.ts

```diff
@@ -17,6 +17,11 @@ hairLength():number {
   return this.bb!.readInt32(this.bb_pos);
```

Comment:
shouldn't this (and similar files) sit parallel to `tests/union_vector/rapunzel.js` ?